schema/deploy/shipping/views.sql

```diff
@@ -3195,12 +3214,16 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         and testing_trigger is null
         -- Filter for postive daily attestations only
         and screen_positive
-        -- Filter for participants who have not tested positive in the past week
-        and prev_pos is not true
+        -- Filter for participants who have not tested positive with us in the past 90 days
+        and (latest_positive_hcov19_collection_date is null or latest_positive_hcov19_collection_date < current_date - interval '90 days')
+        -- Filter for participants who have not tested positive somewhere else in the past 90 days
+        and (latest_prior_test_positive_date is null or latest_prior_test_positive_date < current_date - interval '90 days')
```

Comment:
To do that, we would have to include those columns in the queries that make it to the union at the end of the view. I don't want this view to return COVID-19 positivity info because it would then need to be restricted. What I could do is modify those 5 queries to include the columns that we want to filter for and then in the union select specific columns instead of *.
This would be a touch of work, but I do prefer selecting specific columns in the final query that unions, so I'll do it.

Reply:
I made this change.